### ISSUE TITLE
ChangePasswordForm Validation Message

### ISSUE THREAD

**Issue by nglasl** [silverstripe/silverstripe-framework#4050]
https://github.com/silverstripe/silverstripe-framework/pull/2978

It looks like the solution to this issue was proposed, however never implemented.

**Comment by dhensby** [comment]
Change it in what way?

**Comment by NGlasl** [comment]
Simon pointed out that Form->sessionMessage() only takes two arguments in 3.1, however this doesn't look to be the case from what I can see (http://api.silverstripe.org/3.1/source-class-Form.html#1061). The problem is that the list of password validator messages end up being escaped.

**Comment by normann** [comment]
#2978 is closed, but the bug is still there.

**Comment by nglasl** [comment]
That is correct.

**Comment by dhensby** [comment]
Ok, I've reopened the PR so it can be merged. Can someone please provide me with steps to replicate the bug that the PR fixes and I'll investigate.

**Comment by nglasl** [comment]
You basically just need to set up a password validator, from what I remember. The error messages will come back being escaped, rather than nicely formatted.

```php
$validator = new PasswordValidator();
$validator->minLength(6);
$validator->characterStrength(2 ,array(
	'lowercase',
	'uppercase',
	'digits'
));
Member::set_password_validator($validator);
```

**Comment by tractorcow** [comment]
I've added a new fix at https://github.com/silverstripe/silverstripe-framework/pull/4358

**Comment by nglasl** [comment]
This is just waiting on a PR merge.

**Comment by willmorgan** [comment]
Duplicate of #2978; closing.
